scripts/utils/utils.js

```diff
@@ -139,7 +140,7 @@ function parseParams(params) {
  * Renders nano blocks
  * @param parent The parent element
  */
-export function renderNanoBlocks(parent = document.body) {
+export function renderNanoBlocks(parent = document.body, mv = undefined) {
```

Comment:
What is mv? Is it a context?

Reply:
yes, some sort of context, a modelview. I introduced this to extract the logic that was duplicated in the createNanoBlock method. createNanoBlock is the view and updates itself when the modelview change. If the modelview is not passed to renderNanoBlock, it will not be injected in createNanoBlock which will work as before (see renderLowestPrice for instance which is outside of the product card).

Reply:
It allows to write nanoblock that update themselves based on user interaction:

`function renderFeaturedSavings(mv, text = 'Save', percent = '') {
  const root = createTag(
   [ ... ]
  );

  mv.subscribe(() => {
    // update the view using the state stored in mv.model
  });

  return root;
}`